server/deckUtils.go

```diff
@@ -35,24 +29,24 @@ func printCards(cards []UnoCard) {
 func getDeckConfig() ([4]string, map[string]int, map[string]int) {
 	colors := [4]string{"red", "blue", "green", "yellow"}
 
-	standardCardCounts := map[string]int {
-		"zero": 1,
-		"one": 2,
-		"two": 2,
-		"three": 2,
-		"four": 2,
-		"five": 2,
-		"six": 2,
-		"seven": 2,
-		"eight": 2,
-		"nine": 2,
-		"skip": 2,
+	standardCardCounts := map[string]int{
+		"zero":     1,
```

Comment:
Is it possible to force this kind of formatting via tooling instead of doing it manually?